resources/integration_tools/integration_report.R

```diff
@@ -0,0 +1,160 @@
+library(here)
+source(here::here("lib", "opp.R"))
+
+# replace as appropriate, e.g., 
+# /share/data/opp/data/states/{state}/{city}/raw_csv
+path_to_file <- ""
+
+path_to_clean_rds <- ""
```

Comment:
I think you can just replace this with `opp_path`, `opp_clean_path`, etc. (These are defined in `utils.R`.) Hopefully that'll make it a little more modular. Then you can use `params$city` and `params$state` instead of having to manually change the file for each new city.